pom.xml

```diff
@@ -10,8 +10,8 @@
   <name>Fosstars Rating Core</name>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <version.jackson-databind>2.9.0</version.jackson-databind>
-    <version.jackson-dataformat-yaml>2.9.0</version.jackson-dataformat-yaml>
+    <version.jackson-databind>2.10.3</version.jackson-databind>
+    <version.jackson-dataformat-yaml>2.10.3</version.jackson-dataformat-yaml>
```

Comment:
Hmm, do we actually need two properties ? we can manage with a single one right ? ;)

Reply:
We can. Let's keep two properties since the list of properties shows all dependencies in a shorter way.